docs/conf.py

```diff
@@ -23,6 +23,11 @@
 
 settings.configure()
 
+
+source_dirs = [
```

Comment:
I know this is what this was called before but we should change the name:

```suggestion
instr_source_dir = [
```